src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -186,6 +186,12 @@ bool OutputStateMachineEngine::ActionPassThroughString(const std::wstring_view s
 bool OutputStateMachineEngine::ActionEscDispatch(const wchar_t wch,
                                                  const gsl::span<const wchar_t> intermediates)
 {
+    if (wch == L'\\')
+    {
+        // This is presumably the 7-bit string terminator, which is essentially a no-op.
+        return true;
+    }
+

```

Comment:
I just realised now that this should also be checking that _intermediates_ is empty, otherwise it'll be matching sequences with an intermediate - like `ESC sp \` - which it really shouldn't be. It's not that big a deal, and I'll probably end up moving this into the main switch with an actual action code once PR #7304 is merged, but it wouldn't harm to add an intermediate check here in the mean time.